File: apps/problems/max-element/boilerplate-full/function.js
##USER_CODE_HERE##
    
    const input = require('fs').readFileSync('/Users/vishal/Desktop/PROJECTS/algoearth3/apps/problems/max-element/tests/inputs/##INPUT_FILE_INDEX##.txt', 'utf8').trim().split('\n').join(' ').split(' ');
    const size_arr = parseInt(input.shift());
const arr = input.splice(0, size_arr).map(Number);
    const result = maxElement(arr);
    console.log(result);
        

File: apps/problems/Max-Element/boilerplate-full/function.cpp

            #include <stdio.h>
            #include <stdlib.h>
            #include <stdbool.h>
            #include <string.h>
            #include <limits.h>
            
            ##USER_CODE_HERE##
            
            int main() {
                FILE *file = fopen("/dev/problems/Max-Element/tests/inputs/##INPUT_FILE_INDEX##.txt", "r");
                vector<string> lines;
                string line;
                while (fgets(line, sizeof(line), file)) lines.push_back(line);
                fclose(file);
                int size_arr;
  fscanf(file, "%d", &size_arr);
  int* arr[size_arr];
  for (int i = 0; i < size_arr; i++) fscanf(file, "%d", &arr[i]);
                int result = maxElement(arr);
                fprintf(stdout, "%d\n", result);
                return 0;
                }
            